### PR TITLE
create routes and empty components for them

### PR DESCRIPTION
**Pull request by mark-likhtar**
Create routes and empty route components to see site layout.
- CinemasPage (main page with cinemas, search, sorting,  etc.) "/"
- UserPage ( user page with orders, personal information) "/user"
- EditCinema (page, where admin can edit information about cinemas)    
" /admin/cinema"
- EditFilm ( page, where admin can edit information about films) " /admin/cinema"
- EditSession (page, where admin can edit information about sessions) " /admin/session"
- EditUsers (page, where admin can edit information about users) " /admin/user"
- AdminPage (main admin page) "/admin"

### CODE REVIEW

File: src/components/CinemasPage/CinemasPage.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React, {Component} from 'react';
+
+ export default class CinemasPage extends Component {
+     render() {
+         return (
+             <>
+                hi
```
Comment (andrei-kruhliakou-itechart) [inline src/components/CinemasPage/CinemasPage.js]: replace with meaningful text or remove completely

File: src/components/Header/style.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
+$bgColor: #fff;
+$color: #eb1c23;
+
+
+header {
+    -webkit-box-shadow: 2px 2px 6px 1px rgba(0,0,0,0.75);
```
Comment (andrei-kruhliakou-itechart) [inline src/components/Header/style.scss:6]: you can you some tool like autoprefixer for automating such properties

File: src/components/Main/Main.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import CinemasPage from "../CinemasPage/CinemasPage";
+import UserPage from "../UserPage/UserPage";
+import EditCinema from "../EditCinema/EditCinema";
+import EditFilm from "../EditFilm/EditFilm";
+import EditSession from "../EditSession/EditSession";
+import EditUsers from "../EditUsers/EditUsers";
+import AdminPage from "../AdminPage/AdminPage";
+import { Switch, Route } from "react-router-dom";
```
Comment (andrei-kruhliakou-itechart) [inline src/components/Main/Main.js]: use logical grouping for imports:
- third party libraries
- components
- styles
- images

**Review by andrei-kruhliakou-itechart — CHANGES_REQUESTED** [review]
- add components with empty content for all routes, not just empty js files
- spelling error in output bundle filename ("bandle.js")
- add meaningful info in package.json: name, description, author, license

**Review by mark-likhtar — COMMENTED** [review]
improve all errors and and autoprefixer

**Pull request merged**